app/components/multi-select-dropdown.component.scss

```diff
@@ -18,7 +18,15 @@
         &--selected {
             background: $curiousBlue;
             color: white;
-            font-weight: 600;
```

Comment:
no more font weight?

Reply:
Jonathan & I decided we didn't need/want it. LMK if you wanna fight for it. :]

Reply:
I concur